src/hooks/useSections.ts

```diff
@@ -56,6 +59,21 @@ export default function useSections(
     recommendationsSections
   );
 
+  // Remove sections if necessary
+  useEffect(() => {
+    const features = getSearchSuggestionFeatures(Object.values(podsData || {})?.[0]?.request);
+
+    if (zeroStateActiveSections) {
+      if (features.featureDisplayZeroStateRecommendations) {
+        setActiveSections([]);
```

Comment:
Do we need any of the elses? Wouldn't they come from the initial assignment 🤔

Reply:
I think we will need the else to setActiveSections(sections), but we may not need the one for setActiveSections(zeroStateSections), if we don't expect the a/b state to change in each request.

Correct me if I am wrong but 

```
  const [activeSections, setActiveSections] = useState<UserDefinedSection[]>(
    zeroStateActiveSections ? zeroStateSections : sections
  );
```

Only runs once, so if we don't have the else for setActiveSections(sections), the autocomplete will always be empty.

Reply:
I think you are right. Thanks for the explanation